berth_reservations/relay.py

```diff
@@ -0,0 +1,22 @@
+from django.utils.translation import ugettext_lazy as _
+from graphene import relay
+from graphql_jwt.decorators import login_required, superuser_required
+
+from berth_reservations.exceptions import VenepaikkaGraphQLError
+
+
+class AuthorizationNode(relay.Node):
+    @classmethod
+    @login_required
+    @superuser_required
```

Comment:
hey now 👀

Reply:
we only need `login_required` to ensure that the context has user set, other permissions check are explicit in the method

Reply:
Yeap, took a bit to realise. :derp: